README.md

```diff
@@ -35,6 +35,10 @@ let
 in napalm.buildPackage ./. { packageLock = <path/to/package-lock>; }
 ```
 
+### Napalm with Nix flakes
+
+If you want to use Napalm in your flake project, you can do that by adding it to your overlay. There is also a template, that can help you use napalm in your project. You can use it by running `nix flake init -t "github:nix-community/napalm"`.
```

Comment:
> adding it to your overlay

Maybe something like “adding it to your inputs and passing `napalm.overlay` to your Nixpkgs instance”